app/services/new_starter_trainees_service.rb

```diff
@@ -6,10 +6,12 @@ def initialize(census_date)
   end
 
   def call
+    visible_sources = RecordSources::ALL - [RecordSources::HESA_TRN_DATA]
```

Comment:
I did think we could use the `Trainees::Filter` service here (like the main export and the homepage view does). That would do this logic for us 🤷‍♀️

Reply:
Good idea. I also took the liberty to rename `NewStarterTraineesService` to `FindNewStarterTrainees`

Reply:
love it!